### ISSUE TITLE
Update samples to latest Uno.Sdk 5.4

### ISSUE THREAD

**Issue by agneszitte** [unoplatform/Uno.Samples#828]
Now that Uno.Sdk version **5.4** has been released, we need to update the samples to the **latest stable Uno.Sdk 5.4**.

**Migration notes for 5.4 can be found here: [Uno Platform 5.4 Migration Guide](https://github.com/unoplatform/uno/blob/master/doc/articles/migrating-from-previous-releases.md#uno-platform-54)**

> [!IMPORTANT]  
> **Samples with a status of ⏸️ (On hold), which require migration to the single project structure, are not a priority to be updated or worked on at the moment. The complete list of samples is included in the matrix below to provide an overview of all current samples in the Uno.Samples repo and their statuses between release updates.**

## Steps to follow for each sample you are updating:
- Assign your name in the table below before starting work to ensure no duplication of effort.
- If override versions are still present in `Directory.Build.props`, remove them. 
  [They are no longer necessary with Uno.Sdk 5.3 and above.](https://github.com/unoplatform/uno/blob/master/doc/articles/migrating-from-previous-releases.md#single-project-updates)
- Perform a quick sanity check by testing the sample app on all supported platforms.
- If a README is missing at the root of the sample, please create one. 
  Use the [WindowingSample README.md](https://github.com/unoplatform/Uno.Samples/tree/master/UI/WindowingSamples) or the [Counter README.md](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter) as references for format and folder organization.
- If the sample is not listed in [samples.md](https://github.com/unoplatform/Uno.Samples/blob/master/doc/samples.md), ensure it is added.
- Adjust the status and the last current Uno.Sdk version used by the sample in the table below when changes are merged

---

### Status Legend:

- **📋**: To-Do
- **⌛**: In progress
- **👀**: In review
- **🚧**: Needs re-work (CI errors, additional review comments, etc.)
- **🅱️**: Blocked (Issue and comment with the reason)
- **⏸️**: On hold (Not a priority to update older samples at the moment as they need to migrate to the single structure first)
- **✅**: DONE

### ReadMe.md/Samples.md Legend:
- **❌**: MISSING (If "Descr" is mentioned, it means that the sample is listed but missing the related description)
- **✅**: AVAILABLE

| SAMPLE NAME | ASSIGNEE(s) | STATUS | RELATED PR(s) | COMMENT and current Uno.Sdk/Uno version | README.md | Listed in [samples.md](https://github.com/unoplatform/Uno.Samples/blob/master/doc/samples.md) |
|----------------------------|-------------------------|--------|-------------------------------------------|---------|------------|-----------------------|
| [**Commerce App**](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce) | @vatsashah45 & @agneszitte | ✅ | [#831](https://github.com/unoplatform/Uno.Samples/pull/831) <br> [#835](https://github.com/unoplatform/Uno.Samples/pull/835) | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [**Counter**](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter) | @Kunal22shah & @vatsashah45 | ✅ | [#818](https://github.com/unoplatform/Uno.Samples/pull/818) <br> [#829](https://github.com/unoplatform/Uno.Samples/pull/829) | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [**SimpleCalc App**](https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc) | @Kunal22shah & @vatsashah45 | ✅ | [#818](https://github.com/unoplatform/Uno.Samples/pull/818) <br> [#830](https://github.com/unoplatform/Uno.Samples/pull/830) | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [**ToDo App**](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo) | @vatsashah45 & @agneszitte | ✅ | [#832](https://github.com/unoplatform/Uno.Samples/pull/832) <br> [#834](https://github.com/unoplatform/Uno.Samples/pull/834) | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [**TubePlayer App**](https://github.com/unoplatform/Uno.Samples/tree/master/reference/TubePlayer) | @Kunal22shah | ✅ | [#818](https://github.com/unoplatform/Uno.Samples/pull/818) | **Uno.Sdk: 5.4.5**| ✅ | ✅ |
| [Advanced XBind](https://github.com/unoplatform/Uno.Samples/tree/master/UI/AdvancedXBind) | | 🚧 | [#797](https://github.com/unoplatform/Uno.Samples/pull/797) | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Android Custom Camera](https://github.com/unoplatform/Uno.Samples/tree/master/UI/AndroidCustomCamera) | | 📋 | | Uno.Sdk: 5.3.96 | ✅ | ✅ |
| [Authentication OIDC Demo](https://github.com/unoplatform/Uno.Samples/tree/master/UI/Authentication.OidcDemo) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ✅ | ✅ |
| [Auto-Suggest](https://github.com/unoplatform/Uno.Samples/tree/master/UI/AutoSuggestSample) | | 📋 | | Uno.Sdk: 5.3.96 | ❌ | ✅ |
| [Benchmark](https://github.com/unoplatform/Uno.Samples/tree/master/UI/Benchmark) | @eriklimakc  | ✅ | https://github.com/unoplatform/Uno.Samples/pull/841 | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [Bluetooth Explorer](https://github.com/unoplatform/Uno.Samples/tree/master/UI/BluetoothExplorer) | @morning4coffe-dev | 🅱️/⏸️ | [#796](https://github.com/unoplatform/Uno.Samples/pull/796) | _Uno.WinUI: 5.1.87_ - https://github.com/unoplatform/Uno.Samples/pull/796#issuecomment-2281107518 | ❌ | ✅ |
| [Camera Capture UI](https://github.com/unoplatform/Uno.Samples/tree/master/UI/CameraCaptureUI) | @morning4coffe-dev | 🅱️/⏸️ | [#801](https://github.com/unoplatform/Uno.Samples/pull/801) | _Uno.WinUI: 5.1.87_ - https://github.com/unoplatform/Uno.Samples/pull/801#issue-2455773723 | ❌ | ✅ |
| [CardView Migration](https://github.com/unoplatform/Uno.Samples/tree/master/UI/CardViewMigration) | @eriklimakc  | ✅ | https://github.com/unoplatform/Uno.Samples/pull/844 | **Uno.Sdk: 5.4.8** | ✅ | ✅ |
| [ChatGPT](https://github.com/unoplatform/Uno.Samples/tree/master/UI/ChatGPT) | @Kunal22shah & @eriklimakc | ✅ | [#791](https://github.com/unoplatform/Uno.Samples/pull/791) | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [Chat SignalR](https://github.com/unoplatform/Uno.Samples/tree/master/UI/ChatSignalR) | | 📋 | | Uno.Sdk: 5.3.96 | ✅ | ✅ |
| [Chat UI](https://github.com/unoplatform/Uno.Samples/tree/master/UI/ChatUI) | @eriklimakc | ✅ | https://github.com/unoplatform/Uno.Samples/pull/840 | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [Control Library](https://github.com/unoplatform/Uno.Samples/tree/master/UI/ControlLibrary) | | 📋 | | Uno.Sdk: 5.3.96 | ✅ | ✅ |
| [Country Data](https://github.com/unoplatform/Uno.Samples/tree/master/UI/CountryDataSample) | @rajamatt | 🅱️/⏸️ | _Uno.WinUI: 5.1.87_ - [#804](https://github.com/unoplatform/Uno.Samples/pull/804) | https://github.com/unoplatform/Uno.Samples/pull/804#discussion_r1713914679 | ❌ | ✅ |
| [Custom Sorting](https://github.com/unoplatform/Uno.Samples/tree/master/UI/CustomSorting) | | 📋 | | Uno.Sdk: 5.3.96 | ❌ | ✅ |
| [Dual-Screen Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/DualScreenSample) | | 📋 | | Uno.Sdk: 5.3.96 | ❌ | ✅ |
| [EFCore SQLite Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/EFCoreSQLiteSample) | | 📋 | | Uno.Sdk: 5.3.99 | ✅ | ✅ |
| [Embedded Resources](https://github.com/unoplatform/Uno.Samples/tree/master/UI/EmbeddedResources) | | 📋 | | Uno.Sdk: 5.3.96 | ❌ | ✅ |
| [FileSavePicker iOS](https://github.com/unoplatform/Uno.Samples/tree/master/UI/FileSavePickeriOS) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Food Delivery UI](https://github.com/unoplatform/Uno.Samples/tree/master/UI/FoodDeliveryUI) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌ Descr |
| [HomeIdeas UI](https://github.com/unoplatform/Uno.Samples/tree/master/UI/HomeIdeasUI) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌ |
| [Html Controls](https://github.com/unoplatform/Uno.Samples/tree/master/UI/HtmlControls) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Inserting Separators](https://github.com/unoplatform/Uno.Samples/tree/master/UI/InsertingSeparators) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [LiteDB](https://github.com/unoplatform/Uno.Samples/tree/master/UI/LiteDB/LiteDB) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Localization Samples](https://github.com/unoplatform/Uno.Samples/tree/master/UI/LocalizationSamples) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [MVUX](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MVUX) | @Kunal22shah | ✅ | [#826](https://github.com/unoplatform/Uno.Samples/pull/826) | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [Map Control Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MapControlSample) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Maui Embedding](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MauiEmbedding) | @agneszitte | 📋 | | Uno.Sdk: 5.2.139 - I will review the previous PRs for MAUI Embedding first | ✅ | ✅ |
| [Media Gallery](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MediaGallery) | @eriklimakc  | ⌛ |https://github.com/unoplatform/Uno.Samples/pull/843 | Uno.Sdk: 5.4.5 | ✅ | ✅ |
| [Migrate Custom Renderers](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MigrateCustomRenderers) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌ |
| [Migrating Animations](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MigratingAnimations) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Migrating Effects](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MigratingEffects) | | ⏸️ | | - | ❌ | ✅ |
| [Mvux HowTos](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MvuxHowTos) | | ⏸️ | Needs review | _Uno.WinUI: 5.1.87_ | ✅ | ❌ |
| [Native Frame Navigation](https://github.com/unoplatform/Uno.Samples/tree/master/UI/NativeFrameNav) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Native Style Switch](https://github.com/unoplatform/Uno.Samples/tree/master/UI/NativeStylesSwitch) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Navigation](https://github.com/unoplatform/Uno.Samples/tree/master/UI/Navigation) | @eriklimakc | ✅ | https://github.com/unoplatform/Uno.Samples/pull/838 | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [Neumorphism](https://github.com/unoplatform/Uno.Samples/tree/master/UI/Neumorphism) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Onnx Samples](https://github.com/unoplatform/Uno.Samples/tree/master/UI/OnnxSamples) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌Descr |
| [Package Resources](https://github.com/unoplatform/Uno.Samples/tree/master/UI/PackageResources) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Pet Adopt UI](https://github.com/unoplatform/Uno.Samples/tree/master/UI/PetAdoptUI) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Photo Viewer](https://github.com/unoplatform/Uno.Samples/tree/master/UI/PhotoViewer) | | 🅱️/⏸️ | | _Uno.WinUI: 5.1.50_ - https://github.com/unoplatform/Uno.Samples/pull/796#issuecomment-2281107518 | ❌ | ❌ |
| [PongWars](https://github.com/unoplatform/Uno.Samples/tree/master/UI/PongWars) | @Kunal22shah & @agneszitte  | ✅ | [#792](https://github.com/unoplatform/Uno.Samples/pull/792) | **Uno.Sdk: 5.4.5** | ✅ | ✅ |
| [SQLite Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/SQLiteSample) | | 📋 | | Uno.Sdk: 5.3.99 | ✅ | ✅ |
| [ScottPlot](https://github.com/unoplatform/Uno.Samples/tree/master/UI/ScottPlot) | @agneszitte | ✅ | https://github.com/unoplatform/Uno.Samples/pull/842 | **Uno.Sdk: 5.4.8** | ✅ | ✅ |
| [Skeleton Loader Animation](https://github.com/unoplatform/Uno.Samples/tree/master/UI/SkeletonLoaderAnimation) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌ |
| [SkiaSharp Test](https://github.com/unoplatform/Uno.Samples/tree/master/UI/SkiaSharpTest) | | 📋 | | Uno.Sdk: 5.3.108 | ✅ | ✅ |
| [Skottie Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/SkottieSample) | | 📋 | | Uno.Sdk: 5.3.96 | ❌ | ✅ |
| [Splash Screen Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/SplashScreenSample) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [StatusBar Theme Color](https://github.com/unoplatform/Uno.Samples/tree/master/UI/StatusBarThemeColor) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [The Cat API Client](https://github.com/unoplatform/Uno.Samples/tree/master/UI/TheCatApiClient) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Time Entry](https://github.com/unoplatform/Uno.Samples/tree/master/UI/TimeEntry) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Toy Car](https://github.com/unoplatform/Uno.Samples/tree/master/UI/ToyCar) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Travel UI](https://github.com/unoplatform/Uno.Samples/tree/master/UI/TravelUI) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Uno BackgroundWorker](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoBackgroundWorker) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌Descr |
| [Uno Cakes Mobile](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoCakesMobile) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ✅ | ✅ |
| [Uno Contoso](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoContoso) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ✅ | ✅ |
| [Uno Cupertino Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoCupertinoSample) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Uno Ethereum Blockchain](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoEthereumBlockChain) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Uno GoodReads](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoGoodReads) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Uno Islands Sample App](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoIslandsSampleApp) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Uno Material Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoMaterialSample) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Uno Material Toolkit Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoMaterialToolkitSample) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |
| [Uno SQLite OneDrive Invoice Sample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoSQLiteOneDriveInvoiceSample/Demo) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌Descr |
| [Uno Scroll Reveal](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoScrollReveal) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌Descr |
| [Uno SimplePhotos](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoSimplePhotos) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌ |
| [Uno ZIndex](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoZIndex/UnoZIndex) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌ |
| [WCTDataTreeTab (WCT DataGrid + TreeView +TabView)](https://github.com/unoplatform/Uno.Samples/tree/master/UI/WCTDataTreeTabSample) | @agneszitte | ⏸️ | | _Uno.WinUI: 5.1.87_ | ✅ | ✅ |
| [WebRTC](https://github.com/unoplatform/Uno.Samples/tree/master/UI/WebRTC) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ✅ | ✅ |
| [Windowing Samples](https://github.com/unoplatform/Uno.Samples/tree/master/UI/WindowingSamples) | | 📋 | | Uno.Sdk: 5.3.99 | ✅ | ✅ |
| [Windows Community Toolkit](https://github.com/unoplatform/Uno.Samples/tree/master/UI/WindowsCommunityToolkit) | @agneszitte | 📋 | | Uno.Sdk: 5.2.161 | ✅ | ✅ |
| [XAML Basics: ListView](https://github.com/unoplatform/Uno.Samples/tree/master/UI/XamlBasics/ListViewSample) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ❌Descr |
| [XamlBrewer UnoApp](https://github.com/unoplatform/Uno.Samples/tree/master/UI/XamlBrewerUnoApp) | | ⏸️ | | _Uno.WinUI: 5.1.87_ | ❌ | ✅ |

**Comment by agneszitte** [comment]
cc/ @jeromelaban, @kazo0, @nickrandolph @MartinZikmund, @Youssef1313 
Just for reference, as I'm maintaining a complete list of samples in the matrix above to provide an overview of all current samples in the Uno.Samples repo, along with their statuses between release updates.

**Comment by agneszitte** [comment]
> cc/ @jeromelaban, @kazo0, @nickrandolph @MartinZikmund, @Youssef1313, @Xiaoy312 Just for reference, as I'm maintaining a complete list of samples in the matrix above to provide an overview of all current samples in the Uno.Samples repo, along with their statuses between release updates.

@jeromelaban I am closing this issue in favor of https://github.com/unoplatform/Uno.Samples/issues/848 now that [Uno.Sdk version **5.5** has been released](https://platform.uno/blog/5-5/).
